ddtrace/debugging/_capture/collector.py

```diff
@@ -88,6 +90,7 @@ def __enter__(self):
     def __exit__(self, *exc_info):
         # type: (ExcInfoType) -> None
         try:
+            # BUG: I think @duration, @return are not passed to condition
```

Comment:
That's right, they are not yet passed in 🙁 .

Reply:
refactored